src/i18n/es.js

```diff
@@ -205,25 +205,28 @@ los usuarios, detecta puntos de fuga y oportunidades de mejora! Si necesitas ayu
     visits_without_email: `Usuarios sin email`,
   },
   reports_partials_campaigns: {
-    campaign_name: `Campaña: `,
-    campaign_state: `Estado de la Campaña: `,
+    campaign_name: `Nombre de la  Campaña: `,
+    campaign_state: `Estado de la Campaña `,
     campaign_subject: `Asunto: `,
     campaign_summary: `Resumen de la Campaña`,
     delivery_rate: `Tasa de entrega`,
     emails_delivered: `Emails Entregados:`,
-    hard_and_soft: `Rebotes hard y soft`,
-    header_title: `Reportes parcial de Campañas`,
+    hard_and_soft: `Rebotes Hard y Soft`,
+    header_title_sent: `Reporte Campaña`,
+    header_title_shipping: `Reporte Parcial`,
     last_click_date: `Último Click:`,
     last_open_date: `Última Apertura:`,
     not_open: `No Abiertos`,
     opened: `Abiertos`,
-    page_description: `Reportes parcial de Campañas`,
-    page_title: `Doppler | Reportes parcial de Campañas`,
+    page_description: `Reporte parcial`,
+    page_title: `Doppler | Reporte parcial`,
+    sent: `Enviada`,
+    shipping: `Enviando`,
```

Comment:
Here it should be `shipped`

Reply:
😕 

`sent : Enviada` or `shipped: Enviada`?
 
`shipping: Enviando` or `shipped: Enviando`?

Reply:
Yes I've noticed that I think that `shipped` is the state for sent, and shipping is the state for `in progress`

Reply:
changed